src/value/serde_yaml.rs

```diff
@@ -21,14 +21,11 @@ impl Value for SerdeYaml {
                 inner.iter().enumerate().map(|(i, v)| (Key::Index(i), v)),
             )),
             SerdeYaml::Mapping(ref inner) => Some(Box::new(inner.iter().map(|(k, v)| {
-                (
-                    Key::String(
-                        to_string(k)
-                            .expect("yaml value to serialize into yaml correctly")
-                            .to_owned(),
-                    ),
-                    v,
-                )
+                let mut k = to_string(k).expect("yaml value to serialize into yaml correctly");
+                if k.ends_with('\n') {
```

Comment:
Is there a chance that these are ever `\r\n` on Windows?

I really don't think that this should be fixed here - if `serde_yaml` returns these, there must be something to it, right?

To me it seems fair that users of SerdeYaml have to deal with this information, and the `treediff` doesn't degrade it.

Reply:
ok, I'm removing this commit :+1: